public/table-sort.js

```diff
@@ -167,12 +167,29 @@ function tableSortJs(testingTableSortJS = false, domDocumentWindow = document) {
       }
     }
 
-    function getTableData(tableRows, columnData, isFileSize, isDataAttribute) {
+    function getColSpanData(sortableTable, colSpanData, colSpanSum){
+      sortableTable.querySelectorAll("th").forEach(
+        (th,index) => {
+          colSpanData[index] = th.colSpan
+          if(index===0)
+            colSpanSum[index] = th.colSpan
+          else
+            colSpanSum[index] = colSpanSum[index-1] + th.colSpan
+        })
+    }
+
+
+    function getTableData(tableRows, columnData, isFileSize, isDataAttribute, colSpanData, colSpanSum) {
```

Comment:
I think it's getting to the point where there are that many parameters it makes sense to change this function to take an object containing those parameters,  so then the order of parameters doesn't matter because there would only be one.

Reply:
I mean we could send all the parameters in an object like { tablerows:{...}, columnData:{...},....} like, this. I think it would be appropriate to refactor the code such as that in another PR, letting this PR be only to fixing the colSpan issue.

Reply:
Sure we can defer refactoring code to a different PR.